lib/devise/models/authenticatable.rb

```diff
@@ -152,7 +152,7 @@ def devise_mailer
       #         # If the record is new or changed then delay the
       #         # delivery until the after_commit callback otherwise
       #         # send now because after_commit will not be called.
-      #         if new_record? || changed?
+      #         if new_record? || saved_changes?
```

Comment:
WDYT about leaving a comment stating that `saved_changes?` is for Rails 6 only?
Something like:

```ruby
# ...
# send now because after commit will not be called.
# For Rails < 6 is `changed?` instead of `saved_changes?`.
if new_record? || saved_changes?
```